blurb/blurb.py

```diff
@@ -766,9 +778,23 @@ def find_editor():
             found_path = shutil.which(fallback)
         if found_path and os.path.exists(found_path):
             return found_path
+
+    # For simplicity, recommend the old EDITOR environment varibale. Advanced
+    # users should consult the documention if they want to use more specific
+    # confguration.
     error('Could not find an editor! Set the EDITOR environment variable.')
 
 
+def git_core_editor():
```

Comment:
I generally prefer functions be defined earlier in the file than they are used.  I'm not sure it's worth factoring this out to its own function anyway, since it's only called in one place.